packages/grid/x-grid/src/tests/rows.XGrid.test.tsx

```diff
@@ -342,5 +342,40 @@ describe('<XGrid /> - Rows', () => {
       const isVirtualized = apiRef!.current!.getState().containerSizes!.isVirtualized;
       expect(isVirtualized).to.equal(false);
     });
+
+    describe('Pagination', () => {
```

Comment:
We have dedicated pagination tests. Would it fit better there than here?

Reply:
no because I'm testing the virtualisation with pagination